tor/core/inbox.py

```diff
@@ -58,6 +58,9 @@ def forward_to_slack(item: InboxableMixin, cfg: Config) -> None:
 
 
 @beeline.traced(name="process_reply")
```

Comment:
This must be directly above the `def process_reply` line, it's an annotation.

I recommend moving the extract function further up.
A style guideline is also to have two empty lines between function definitions